packages/ckeditor5-package-generator/lib/templates/ts/package.json

```
@@ -80,14 +81,16 @@
     "typescript": "^4.7.4"
   },
   "scripts": {
+    "build": "tsc -p ./tsconfig.release.json",
     "dll:build": "ckeditor5-package-tools dll:build",
     "dll:serve": "http-server ./ -o sample/dll.html",
     "lint": "eslint \"**/*.{js,ts}\" --quiet --ignore-pattern \"build/\"",
     "start": "ckeditor5-package-tools start",
     "stylelint": "stylelint --quiet --allow-empty-input 'theme/**/*.css'",
     "test": "ckeditor5-package-tools test",
     "prepare": "<%= program %> run dll:build",
-    "prepublishOnly": "<%= program %> run dll:build",
+    "prepublishOnly": "<%= program %> run dll:build && <%= program %> run build && ckeditor5-package-tools updatePkgJsonMain js",
```

```suggestion
    "prepublishOnly": "<%= program %> run dll:build && <%= program %> run build && ckeditor5-package-tools export-package-as-javascript",
```

Perhaps the name of the task is subject to change.  The goal is to have a single task that defines what it does. 

The current proposal does not explain too much. Also, arguments/options/modifiers in CLI uses kebab-case.